mkdocs.yml

```diff
@@ -140,3 +140,4 @@ plugins:
         'ftldns/database.md': database/index.md
         'ftldns/regex/index.md': ftldns/regex/overview.md
         'main/presentations.md': index.md
+        'main/prerequesites.md': main/prerequisites.md
```

Comment:
Can we split this out to a separate PR please? Would save me from hunting down all the places we use the misspelled link...

Reply:
See #331